docs/en/stack/ml/anomaly-detection/ootb-ml-jobs-siem.asciidoc

```diff
@@ -528,6 +528,153 @@ Required ECS fields when not using {beats}:::
 
 // end::siem-auditbeat-auth-jobs[]
 
+[discrete]
+[[security-authentication]]
+== Security: Authentication
+
+Detect anomalous activity in your ECS-compatible authentication logs.
+
+These configurations are only available if data exists that matches the 
+recognizer query specified in the
+https://github.com/elastic/kibana/blob/{branch}/x-pack/plugins/ml/server/models/data_recognizer/modules/security_auth/manifest.json#L8[manifest file].
+
+If you create these jobs in the *{ml-app}* app, you must first create the
```

Comment:
This statement (and it is repeated below) is not correct. The ML UI runs the query in the `manifest.json` against the index pattern (or 'data view' from 8.0) to decide whether to display the card in the job wizard (so actually the statement you have for the security app is the one to use when discussing whether the jobs appear in the ML app).

Reply:
I've taken another run at this statement in 33d2596b3801340540626e42ecab5a4bd5748e7c

Reply:
This change stating how you create the jobs in the ML app LGTM